green_cli/liquid.py

```diff
@@ -53,8 +70,12 @@ class Asset(click.ParamType):
     name = 'asset'
 
     def convert(self, value, param, ctx):
-        assert 'asset_tag' not in ctx.params['details']['addressees'][-1]
-        ctx.params['details']['addressees'][-1]['asset_tag'] = value
+        assert 'asset_id' not in ctx.params['details']['addressees'][-1]
+        if not _get_network()['mainnet']:
```

Comment:
Maybe add a comment on why this is not enabled in mainnet

Reply:
Done, cheers